storage/pv-to-path-mapping

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# map PVS to storage paths
+printf "%-40s\t%-40s\t%s\n" "PATH" "PV" "CLAIM"
+PVS=$(oc get pv -o jsonpath='{.items[*].metadata.name}' | tr ' ' "\n")
+declare -A PATH_TO_PV_MAP
+while read pv <&3; do
+  path=$(oc describe pv ${pv} | grep Path: | awk '{print $2}')
+  claim=$(oc describe pv ${pv} | grep Claim: | awk '{print $2}')
```

Comment:
Think jsonpath can be used here:
`jsonpath="{.spec.claimRef.name}"`

Reply:
doh, i thought i caught all those

Reply:
fixed.